{"name": "test_101_110[全部输入，输入密码不正确]", "status": "passed", "parameters": [{"name": "username", "value": "'17854225747'"}, {"name": "password", "value": "'12345'"}, {"name": "verify_code", "value": "'8888'"}, {"name": "expectedtext", "value": "'密码错误!'"}], "start": 1596270210817, "stop": 1596270235578, "uuid": "cca7b329-638c-4c20-9d1c-ba10056284af", "historyId": "30a08fde41906cd14bc091a8d73e4a82", "testCaseId": "674c8a70a912fd8799a90691a3422221", "fullName": "cases.登录.test_login.Test_login#test_101_110", "labels": [{"name": "story", "value": "登录失败测试"}, {"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "cases.登录"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "PC-20180809BGFX"}, {"name": "thread", "value": "17008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.登录.test_login"}]}